components/esp_system/port/soc/esp32/clk.c

```diff
@@ -28,7 +28,8 @@
 #else
 #include "stubs.h"
 #define CONFIG_ESP32_DEFAULT_CPU_FREQ_MHZ   ESP_SOC_DEFAULT_CPU_FREQ_MHZ
-#endif
+#define CONFIG_ESP32_RTC_CLK_CAL_CYCLES ESP_SOC_DEFAULT_RTC_CLK_CAL_CYCLES
```

Comment:
`#define CONFIG_ESP32_RTC_CLK_CAL_CYCLES ESP_SOC_DEFAULT_RTC_CLK_CAL_CYCLES` is already being defined in PM PR in Zephyr's upstream. Can you remove this entry from all `clk.c` files?